python/lsst/ts/externalscripts/maintel/take_twilight_flats_lsstcam.py

```diff
@@ -0,0 +1,188 @@
+            properties:
+              filter:
+                description: Filter name or ID.
+                anyOf:
+                  - type: string
+                  - type: integer
+                    minimum: 1
+                  - type: "null"
+            required: ["filter"]
+            additionalProperties: false
+        """
+        schema_dict = yaml.safe_load(schema_yaml)
+
+        base_schema_dict = super(TakeTwilightFlatsLSSTCam, cls).get_schema()
+
+        for prop in base_schema_dict["properties"]:
+            schema_dict["properties"][prop] = base_schema_dict["properties"][prop]
+
+        return schema_dict
+
+    async def get_sky_counts(self) -> float:
+        """Abstract method to get the median sky counts from the last image.
+
+        Returns
+        -------
+        float
+            Sky counts in electrons.
+        """
+        # TODO: query consDB with util once such the LSSTCam schema exists
+        sky_counts = 0
+        return sky_counts
+
+    def get_instrument_name(self) -> str:
+        """Get instrument name.
+
+        Returns
+        -------
+        instrument_name: `string`
+        """
+        return "LSSTCam"
+
+    def get_instrument_configuration(self) -> dict:
+        return dict(
+            filter=self.config.filter,
+        )
+
+    def get_instrument_filter(self) -> str:
+        """Get instrument filter configuration.
+
+        Returns
+        -------
+        instrument_filter: `string`
+        """
+        return f"{self.config.filter}"
+
+    async def track_radec_and_setup_instrument(self, ra, dec):
+        """Method to set the instrument filter and slew to desired field.
+
+        Parameters
+        ----------
+        ra : float
+            RA of target field.
+        dec : float
+            Dec of target field.
+        """
+        current_filter = await self.comcam.get_current_filter()
+
+        self.tracking_started = True
+
+        if current_filter != self.config.filter:
+            self.log.debug(
+                f"Filter change required: {current_filter} -> {self.config.filter}"
+            )
+            await self._handle_slew_and_change_filter()
+        else:
+            self.log.debug(
+                f"Already in the desired filter ({current_filter}), slewing and tracking."
+            )
+
+        await self.mtcs.slew_icrs(
+            ra=ra,
+            dec=dec,
+            rot_type=RotType.PhysicalSky,
+        )
+
+    async def _handle_slew_and_change_filter(self):
+        """Handle slewing and changing filter at the same time.
+
+        For ComCam (and MainCam) we need to send the rotator to zero and keep
+        it there while the filter is changing.
+        """
+
+        tasks_slew_with_fixed_rot = [
+            asyncio.create_task(
+                self.mtcs.slew_icrs(
+                    ra=self.config.ra,
+                    dec=self.config.dec,
+                    rot_type=RotType.PhysicalSky,
```

Comment:
This one needs to be `RotType.Physical` instead. this will ensure the rotator is kept at the fixed position.